homeassistant/components/http/view.py

```diff
@@ -1,180 +1,11 @@
 """Support for views."""
 from __future__ import annotations
 
-import asyncio
-from collections.abc import Awaitable, Callable
-from http import HTTPStatus
 import logging
-from typing import Any
 
-from aiohttp import web
-from aiohttp.typedefs import LooseHeaders
-from aiohttp.web_exceptions import (
-    HTTPBadRequest,
-    HTTPInternalServerError,
-    HTTPUnauthorized,
+from homeassistant.helpers.http import (  # noqa: F401
+    HomeAssistantView,
+    request_handler_factory,
 )
-from aiohttp.web_urldispatcher import AbstractRoute
-import voluptuous as vol
-
-from homeassistant import exceptions
-from homeassistant.const import CONTENT_TYPE_JSON
-from homeassistant.core import Context, HomeAssistant, is_callback
-from homeassistant.helpers.json import (
-    find_paths_unserializable_data,
-    json_bytes,
-    json_dumps,
-)
-from homeassistant.util.json import JSON_ENCODE_EXCEPTIONS, format_unserializable_data
-
-from .const import KEY_AUTHENTICATED
 
 _LOGGER = logging.getLogger(__name__)
```

Comment:
Is the logger still needed?

Reply:
removed in d21a24ca76de501e037f7f55e277f0e13f0c3e2b